zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -471,6 +474,7 @@ ascan.options.attackPrompt.label = In Attack Mode prompt to rescan nodes when sc
 ascan.options.attackRescan.label = In Attack Mode always rescan nodes when scope changed.
 ascan.options.delayInMs.deprecated = Note: The "Delay When Scanning" has been deprecated. See help for more details.
 ascan.options.delayInMs.label = Delay When Scanning (In Milliseconds):
+ascan.options.excludeanticsrf.label = Exclude anti-CSRF tokens.
```

Comment:
This confused me when I saw it as we have:

- [x] Handle anti-CSRF tokens.
- [x] Exclude anti-CSRF tokens.

How about:

- [x] Handle anti-CSRF tokens in forms.
- [x] Exclude anti-CSRF tokens from attacks.